about.html

```diff
@@ -22,24 +21,19 @@
 <h3>Applications</h3>
 
 <ul>
-<li>PcManFm-qt - File Manager</li>
```

Comment:
This is not how to spell https://github.com/lxqt/pcmanfm-qt

Reply:
I'm afraid there are much other combinations around, but true.